packages/snaps-controllers/src/multichain/MultiChainController.test.ts

```diff
@@ -1,10 +1,7 @@
 /* eslint-disable jest/prefer-strict-equal */
 
-import {
-  fromEntries,
-  getSnapPermissionName,
-  getSnapSourceShasum,
-} from '@metamask/snaps-utils';
+import { targetKey as permissionKey } from '@metamask/rpc-methods/src/restricted/invokeSnap';
```

Comment:
Can this be exported under the root, with a name that makes more sense for a constant?

```suggestion
import { PERMISSION_KEY } from '@metamask/rpc-methods';
```